src/safeposix/syscalls/net_calls.rs

```diff
@@ -3167,21 +3252,26 @@ impl Cage {
                             }
                         };
                         ret_addr.set_addr(addr);
+                        // sets port to 0
```

Comment:
Why have this comment?  Probably should explain why you're doing it or just omit it.